src/components/LocationDetails.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/LocationDetails.css";
+
+function LocationDetails(props) {
+  const { city, country, errorMessage } = props;
```

Comment:
You could destructure this directly in the function definition instead of using the props object:

```
function LocationDetails({ city, country, errorMessage }) {
```